### ISSUE TITLE
A discrete CEM3320 board that doesn't need a CEM3320 would be nice. 

### ISSUE THREAD

**Issue by 01GOD** [JordanAceto/VCF_plug_in_boards#6]
Hi! Thanks for posting those! I like that mobo plugin design. 

A discrete CEM3320 board that doesn't need a CEM3320 would be nice.

**Comment by JordanAceto** [comment]
Hi there, just noticed this. Interesting idea. I am not familiar with any discrete CEM3320 projects, do you have any references to designs you're thinking of here?

The SSM2040 has been sort-of-reverse-engineered by Jurgen Haible a long time ago: http://jhaible.com/legacy/tonline_stuff/jh2040.gif (schem has obvious error with reversed PNP current mirrors but otherwise is ok). I started, but have not finished a version of this project. This is basically the same as the MOTM-440 with the control range scaled to my [0v, 2v] range.

I don't know of a similar effort to reverse engineer the CEM3320, and this isn't something I am going to put time into at the moment. Since afaik the CEM3320 is pretty similar internally to cascaded LM13700 OTA stages you could probably get reasonably close sound-wise with a couple LM13700s and an expo converter stage. But then you've strayed pretty far from CEM land. Digging into the guts and replacing the OTA stages with discrete BJT/FET equivalents would be a pretty big task.

In short, if you have more info about what a discrete CEM3320 might look like I'd consider giving it a shot at some point, but won't promise much. Since we can buy the Alpha CEM3320 clones cheaply right now there is not a ton of motivation to make a discrete version.

Cheers :)
